TTS/tts/utils/text/japanese/phonemizer_test.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
this file needs to go to `tests/` folder.

Reply:
Tested with `make test`

[tts_test.txt](https://github.com/coqui-ai/TTS/files/6580943/tts_test.txt)